fizzbuzz.py

```diff
@@ -0,0 +1,20 @@
+#function fizzbuzz
+def fizzbuzz(a, b):
+    if isinstance((a and b), list):
```

Comment:
This is a bit buggy as it will fail in some cases. For example, try to run your program passing in `True` and a list i.e `fizzbuzz(True, [])`. Your program will break. I think it will be better if you checked them individually if they are all lists. The most simple solution will be :-
```python
def fizzbuzz(a, b):
    if isinstance(a, list) and isinstance(b, list):
        combined_length = len(a) + len(b)
```

Reply:
This is awesome, will apply this